src/main/java/com/quimify/api/health/HealthController.java

```diff
@@ -0,0 +1,30 @@
+package com.quimify.api.health;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/health")
+public class HealthController {
+
+    @Autowired
+    HealthIndicator healthIndicator;
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    public HealthController(QuimifyHealthIndicator healthIndicator) {
+        this.healthIndicator = healthIndicator;
+    }
+
+    @GetMapping
+    public Health getHealth() {
```

Comment:
If you don't want to return `String`, then return `HealthResult` instead. Use `Health` only in `HealthService`.
Classes like `HealthResult` are meant to be sent to the client, and are not meant to be used internally

Reply:
Okey! In addition, I also used @Autowired for HealthService.